hooks/README

```diff
@@ -0,0 +1,2 @@
+The hooks directory represents build phase hooks for Dockerhub builds. 
+https://docs.docker.com/docker-hub/builds/advanced/
```

Comment:
This is missing a newline (github shows an error marker).

Reply:
Added newline.